### ISSUE TITLE
MRMS from NCSS has both missing_value and _FillValue

### ISSUE THREAD

**Issue by dopplershift** [Unidata/thredds#1050]
When requesting data from MRMS using NCSS (e.g. 
http://thredds.ucar.edu/thredds/ncss/grib/NCEP/MRMS/BaseRef/MRMS_BaseReflectivity_20180302_1600.grib2?var=MergedBaseReflectivityQC_altitude_above_msl&horizStride=1&time_start=2018-03-02T16%3A00%3A17Z&time_end=2018-03-02T16%3A58%3A17Z&timeStride=1&vertCoord=&accept=netcdf4) you end up with a file that has both `missing_value` and `_FillValue`:

```
	float MergedBaseReflectivityQC_altitude_above_msl(time, altitude_above_msl, lat, lon) ;
		MergedBaseReflectivityQC_altitude_above_msl:long_name = "MergedBaseReflectivityQC @ Specific altitude above mean sea level" ;
		MergedBaseReflectivityQC_altitude_above_msl:units = "dBZ" ;
		MergedBaseReflectivityQC_altitude_above_msl:description = "Mosaic Base Reflectivity (optimal method)" ;
		MergedBaseReflectivityQC_altitude_above_msl:missing_value = -99.f ;
		MergedBaseReflectivityQC_altitude_above_msl:_FillValue = -999.f ;
		MergedBaseReflectivityQC_altitude_above_msl:grid_mapping = "LatLon_Projection" ;
		MergedBaseReflectivityQC_altitude_above_msl:coordinates = "reftime time altitude_above_msl lat lon " ;
		MergedBaseReflectivityQC_altitude_above_msl:Grib_Variable_Id = "VAR_209-11-0_L102" ;
		MergedBaseReflectivityQC_altitude_above_msl:Grib2_Parameter = 209, 11, 0 ;
		MergedBaseReflectivityQC_altitude_above_msl:Grib2_Parameter_Name = "MergedBaseReflectivityQC" ;
		MergedBaseReflectivityQC_altitude_above_msl:Grib2_Level_Type = 102 ;
		MergedBaseReflectivityQC_altitude_above_msl:Grib2_Level_Desc = "Specific altitude above mean sea level" ;
		MergedBaseReflectivityQC_altitude_above_msl:Grib2_Generating_Process_Type = "Observation" ;
```

This seems odd and is breaking xarray.

**Comment by lesserwhirls** [comment]
In this case, MRMS has both `missing` and a `no coverage` parameter in the GRIB2 [parameter table](https://github.com/Unidata/thredds/blob/master/grib/src/main/resources/resources/grib2/mrms/UserTable_MRMS_v10.0.1.csv). `missing` is used for the `missing_value` attribute, and `no coverage` is used for `_FillValue` in the translation of GRIB into the CDM by the GRIB IOSP.

`_FillValue` is what applications first look for in terms of invalid data, but has to be a scalar, so we have to also use `missing_value` in this case. As a side note, `missing_data` more flexible and can be a vector, according to the [NUG](https://www.unidata.ucar.edu/software/netcdf/docs/attribute_conventions.html) attribute conventions).

I don't see anything in the [CF documentation](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/cf-conventions.html#missing-data) on missing data that would prohibit the use of both, so perhaps this is worth supporting in xarray?

**Comment by dopplershift** [comment]
It might be worth bringing up with xarray, but they specifically catch and error out on this case, so I assume there's a reason.

**Comment by dopplershift** [comment]
We'll fix it over there--see pydata/xarray#1749
